shpc/main/registry/filesystem.py

```diff
@@ -99,25 +110,13 @@ def find(self, name):
         if os.path.exists(container_yaml):
             return FilesystemResult(name, container_yaml)
 
-    def cleanup(self):
```

Comment:
So this was moved elsewhere? Someone (e.g., a developer user) might still want this ability.

Reply:
Entirely delete a local registry ? really ? I removed it because I couldn't understand the use-case, but I can drop the commit if needed

Reply:
I think the interface between providers should be consistent, yes. If someone clones a remote to a temporary directory to do some work, instantiating it as a Filesystem registry, after they should be able to clean it up.